### ISSUE TITLE
For resellers: overview of all sites by the same owner and form to buy more sites

### ISSUE THREAD

**Issue by ottok** [Seravo/seravo-plugin#31]

**Comment by l3ku** [comment]
I personally somehow don't like the idea that resellers should manage their own site and the sites of their customers in their own WP admin backend via the Seravo Plugin. There is a need for a completely separate admin area where resellers (and in the future regular customers) could purchase sites, as it sounds clumsy to me that there already needs to be an existing site to purchase another site, but all sites could be purchased from a single area (even the first one?). This way we could still include the single site administration-related functionality in Seravo Plugin (like logs, updates and reports), but do the reseller administration elsewhere (domains, DNS, additional services). This could also help with whitelabeling, as the reseller would not have to give a link to a separate admin area and thus we would not have to hide as much functionality from the Seravo Plugin on the sites of our reseller's customers.

**Comment by ottok** [comment]
What if we used an approach similar to ManageWP where one WP installation (the reseller's main site) talks to all the other sites and shows an overview, all based on stuff going on in the PHP code?

**Comment by EddieFin** [comment]
ManageWP is the GoDaddy owned site management SaaS tool. MainWP is probably the one that works in that way (don't know for sure).

In my opinion (and based on reviews of some of the competition), a separate dashboard is not considered annoying or confusing, it always seems to be a plus when it's done right. 

Not talking about cPanel or anything here, but just Googling reviews of the competition and reading through those gives a pretty good idea how much people (and the reviewers, who we'll need to get our product in front of too) seem to love the dashboards.

**Comment by EddieFin** [comment]
And just to throw the idea our here, there are some pretty solid looking and easily customizable react-based dashboard templates that are affordable and would give us a huge jumpstart on building one, since we wouldn't have to do the visual side of things (or at least most of it is included in the template).

Example: https://dashboardpack.com/theme-details/architectui-dashboard-react-pro/

**Comment by annijpesonen** [comment]
I agree with Leo and Eetu on this that the panel really needs to be separate. This opinion is based on both my impressions reading about different hosting solutions and their popularity online but also on the discussions we've had about the panel in the first two workshop wednesdays. 

The resellers not only need to get an overview of the functionality of all their client's websites, but to be able to manage everything there is to manage related to reselling, such as ordering new and cancelling old services and getting an overview of their invoicing situation as a whole and from each site separately (when the next invoice will be due, is there something unpaid, what are the total sums etc.). I don't feel confident having these kind of integrations to invoicing systems in WordPress. Also, I would find it hard to determine which is the reseller's main site, because not always do they have their own site hosted with us, and it sounds unlikely to me that they would want their every other customer site to be linked to that either.

Then there is also the question that should we separate the access to the control panel only to different customers. I agree that having the Seravo plugin is beneficial to some (smaller) customers, but it would feel a bit funny from a reseller's point of view that they would make some changes and see some things from the separate admin panel and then have to go and do some other things via a separate plugin such as Seravo plugin (naturally some of them use the command line instead and not dependent on if there is another dashboard or not, but not all).
